jekyll/_cci2/credits.md

```diff
@@ -53,7 +52,7 @@ Upgrading to a performance plan offers several improvements over the free plan:
 
 - Access to all machine sizes for Docker/Linux based machines
 - Access to medium sized MacOS machines (4 CPUs, 8gb Ram at 50 credits/minute)
-- Unlimited user seat count (at $15 for the first three users and $15 for each additional user)
+- Unlimited user seat count (at 25,000 credits for the first three users and 25,000 credits for each additional user)
```

Comment:
Thoughts if we say "scalable" or "flexible" instead of "unlimited"? I think saying unlimited but putting a price might sound a little contradictory.

Reply:
fixed